src/makeRobot.js

```diff
@@ -39,6 +39,62 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
```

Comment:
```suggestion
    },
    
    get info() {
```
Добавляй пустые строки между методами, так код легче читать